web/sites/default/config/core.entity_view_display.node.park_facility.default.yml

```diff
@@ -60,7 +60,7 @@ content:
     settings:
       link: false
     third_party_settings: {  }
-    weight: 19
+    weight: 20
```

Comment:
The default display doesn't appear to have meaningful changes. The order is the same as before even though the weights are modified. Can we revert this file to the previous version?